pyfr/backends/base/backend.py

```diff
@@ -37,6 +37,8 @@ def __init__(self, cfg):
 
         # Convert to a NumPy data type
         self.fpdtype = np.dtype(prec).type
+        self.fpdtype_eps = np.finfo(self.fpdtype).eps
```

Comment:
Don't know if we still need eps since we solve for `exp(\zeta) \in [0,1]` now

Reply:
Yeah, it's not currently used.  Although a couple of pending PR's want it, so decided to add whilst I was working on the code.